spec/javascripts/support/mocha.js

```diff
@@ -0,0 +1,6 @@
+const babelRegister = require('@babel/register');
+
+babelRegister({ ignore: [/node_modules\/(?!@18f\/identity-)/] });
+
+// Ignore SCSS imports.
```

Comment:
maybe leave a comment that they're only used for dependencies of css packs?

Reply:
> maybe leave a comment that they're only used for dependencies of css packs?

👍 

Clarified the comment in the rebased 1b59c2e.